lib/transformer.dart

```diff
@@ -84,14 +85,14 @@ class InitializeTransformer extends Transformer {
     if (primaryId == null) primaryId = transform.primaryInput.id;
     var newEntryPointId = new AssetId(primaryId.package,
         '${path.url.withoutExtension(primaryId.path)}.initialize.dart');
-    return transform.hasInput(newEntryPointId).then((exists) {
+    return transform.hasInput(newEntryPointId).then((exists) async {
```

Comment:
The error was:
```
[error] The return type 'Future<AssetId>' is not a 'AssetId', as defined by the method ''. (package:initialize/transformer.dart, line 95, col 24)
```

I think that is related to the `Future` flattening thing. Probably `Future<T>.then` should return a `T` and the trick of returning a `Future<T>` is not possibile in pure `strong` mode ?